include/common_structs.h

```diff
@@ -542,7 +543,8 @@ typedef struct BattleStatus {
     /* 0x17E */ s16 currentAttackDamage;
     /* 0x180 */ s16 lastAttackDamage;
     /* 0x182 */ char unk_182[2];
-    /* 0x184 */ s32 currentTargetListFlags; /* set when creating a target list, also obtain from the flags field of moves */
+    /* 0x184 */ s32
+        currentTargetListFlags; /* set when creating a target list, also obtain from the flags field of moves */
```

Comment:
This is awkward, we could set the clang-format line length limit to `999`, or manually place the documentation comment above.

Reply:
Yeah this isn't ideal. We disable the formatting of headers in oot for this kind of reason.